# object_localizer

MAP_FRAME_ID: "map"
HZ: 10

# # Number of Particles
NUM_OF_PARTICLES: 1000

# Initial Position
INIT_X: 7.4
INIT_Y: 0.7
INIT_YAW: 1.5708

# Initial Variance
INIT_X_VAR: 0.3
INIT_Y_VAR: 0.3
INIT_YAW_VAR: 0.2

# Variance Threshold
LOWER_X_VAR_TH: 0.10
LOWER_Y_VAR_TH: 0.10
LOWER_YAW_VAR_TH: 0.05
UPPER_X_VAR_TH: 0.80
UPPER_Y_VAR_TH: 0.80
UPPER_YAW_VAR_TH: 0.80

# Motion Noise (robot-specific error parameters)
ALPHA_1: 0.1
ALPHA_2: 0.1
ALPHA_3: 0.1
ALPHA_4: 0.1

# Resampling Parameters
ALPHA_SLOW: 0.001
ALPHA_FAST: 0.1

# Parameters to update or not to update
DISTANCE_TH: 0.1
ANGLE_TH: 0.1

# Percentage of particles selected
SELECTION_RATE: 0.2

# object_detector_msgs::ObjectPositions filter params
PROBABILITY_TH: 0.8
VISIBLE_LOWER_DISTANCE: 0.2
VISIBLE_UPPER_DISTANCE: 6.5
# ANGLE_OF_VIEW: 1.50

# Observation Update Parameters
DISTANCE_NOISE: 0.05